src/test/java/org/broadinstitute/consent/http/resources/ResearcherResourceTest.java

```diff
@@ -65,107 +61,18 @@ private void initResource() {
-        when(researcherService.setProperties(any(), any())).thenReturn(Collections.emptyList());
-        initResource();
-        Response response = resource.registerProperties(authUser, uriInfo, null);
-        Assert.assertEquals(Response.Status.CREATED.getStatusCode(), response.getStatus());
-    }
-
-    @Test
-    public void testRegisterPropertiesNotFound() {
-        when(researcherService.setProperties(any(), any())).thenThrow(new NotFoundException("User Not Found"));
-        initResource();
-        Response response = resource.registerProperties(authUser, uriInfo, null);
-        Assert.assertEquals(Response.Status.NOT_FOUND.getStatusCode(), response.getStatus());
-    }
-
-    @Test
-    public void testRegisterPropertiesInvalidFields() {
-        when(researcherService.setProperties(any(), any())).thenThrow(new IllegalArgumentException("Fields not valid"));
-        initResource();
-        Response response = resource.registerProperties(authUser, uriInfo, null);
-        Assert.assertEquals(Response.Status.BAD_REQUEST.getStatusCode(), response.getStatus());
-    }
-
-    @Test
-    public void testUpdateProperties() {
-        initResource();
-
-        Response response1 = resource.updateProperties(authUser, false, null);
-        Assert.assertEquals(Response.Status.OK.getStatusCode(), response1.getStatus());
-        Response response2 = resource.updateProperties(authUser, true, null);
-        Assert.assertEquals(Response.Status.OK.getStatusCode(), response2.getStatus());
-    }
-
-    @Test
-    public void testUpdatePropertiesNotFound() {
-        when(researcherService.updateProperties(any(), any(), any())).thenThrow(new NotFoundException("User Not Found"));
+    public void testDeleteAllProperties() {
+        doNothing().when(researcherService).deleteResearcherProperties(any());
         initResource();
-        Response response = resource.updateProperties(authUser, false, null);
-        Assert.assertEquals(Response.Status.NOT_FOUND.getStatusCode(), response.getStatus());
+        Response response = resource.deleteAllProperties(1);
+        assertEquals(200, response.getStatus());
     }
 
     @Test
-    public void testUpdatePropertiesInvalidFields() {
-        when(researcherService.updateProperties(any(), any(), any())).thenThrow(new IllegalArgumentException("Invalid Fields"));
+    public void testDeleteAllPropertiesError() {
+        doThrow(new RuntimeException()).when(researcherService).deleteResearcherProperties(any());
         initResource();
-        Response response = resource.updateProperties(authUser, false, null);
-        Assert.assertEquals(Response.Status.BAD_REQUEST.getStatusCode(), response.getStatus());
+        Response response = resource.deleteAllProperties(1);
+        assertEquals(500, response.getStatus());
     }
-
-    @Test
-    public void testDescribeAllResearcherPropertiesAdmin() {
-        when(researcherService.describeResearcherPropertiesMap(any())).thenReturn(new HashMap<>());
```

Comment:
In removing these, `describeResearcherPropertiesMap` is no longer used by anyone